### PR TITLE
docs: Assembling grips and fpv 

### PR DESCRIPTION
**Pull request by Alamoris**

### CODE REVIEW

**Comment by okalachev** [comment]
Links from other related articles needed.

File: docs/ru/magnetic_grip.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Сборка и настройка элеткромагнитного захвата
+
+Магнитный захват можно собрать различными способами в соответствии с электрической схемой.
+
+<img src="../assets/magnet_grip/scheme.jpg" width=300 class="zoom border center">
```
Comment (okalachev) [inline docs/ru/magnetic_grip.md:5]: File is missing.

File: docs/ru/magnetic_grip.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+1. На паечной плате разместите *диод Шоттки*, *резистор 10кОм* и *транзистор*.
+
+    <img src="../assets/magnet_grip/magnet1.png" width=300 class="zoom border center">
+
+2. Припаяйте контакты с другой стороны платы и откусите оставшиеся ножки элементов.
+
+3. Соедините контакты резистра и двух крайних ножек транзистра.
+
+    <img src="../assets/magnet_grip/magnet2.png" width=300 class="zoom border center">
```
Comment (okalachev) [inline docs/ru/magnetic_grip.md:19]: Here and further: files are missing.

**Review by okalachev — CHANGES_REQUESTED** [review]

File: docs/ru/magnetic_grip.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+Пример кода активирующего магнитный захват, можно посмотреть [тут](gpio.md#подключение-электромагнита).
+
+## Подключение к Arduino
+
+Для управления захватом в ручном режиме его можно подключить к плате *Arduino Nano*.
+
+Удобно ее располагать на той же паечной плате. Для этого вставьте ее в подходящие отверстия и припаяйте с обратной стороны к плате.
+
+<img src="../assets/magnet_grip/magnet_adruino1.png" width=300 class="zoom border center">
```
Comment (okalachev) [inline docs/ru/magnetic_grip.md:55]: Note: the file should spell arduino, not "adruino1".

Comment (okalachev) [inline docs/ru/magnetic_grip.md:55]: Note: the files should spell arduino, not "adruino1".

**Review by okalachev — CHANGES_REQUESTED** [review]

**Comment by Alamoris** [comment]
Added images and fixed naming errors

**Comment by okalachev** [comment]
@Alamoris, add links to these articles from whatever assembly-related articles, when they are needed.

**Pull request merged**